src/poprox_storage/repositories/accounts.py

```diff
@@ -94,15 +94,17 @@ def store_account(self, account: Account) -> UUID | None:
         row = self.conn.execute(query).one_or_none()
         return row.account_id
 
-    def store_new_account(self, email: str, source: str) -> Account:
+    def store_new_account(self, email: str, source: str, sub_source: str = None) -> Account:
         account_tbl = self.tables["accounts"]
         query = (
             sqlalchemy.insert(account_tbl)
-            .values(email=email, source=source, status="new_account")
+            .values(email=email, source=source, sub_source=sub_source, status="new_account")
             .returning(account_tbl.c.account_id, account_tbl.c.email, account_tbl.c.status)
         )
         row = self.conn.execute(query).one_or_none()
-        return Account(account_id=row.account_id, email=row.email, status=row.status, source=source)
+        return Account(
+            account_id=row.account_id, email=row.email, status=row.status, source=source, sub_source=sub_source
```

Comment:
This looks okay, but I don't see [a subsource field](https://github.com/CCRI-POPROX/poprox-concepts/blob/main/src/poprox_concepts/domain/account.py#L11) on the `Account` class yet.